tests/integration/product-type-import.spec.js

```diff
@@ -0,0 +1,174 @@
+import { SphereClient } from 'sphere-node-sdk'
+import Promise from 'bluebird'
+import test from 'tape'
+import ProductTypeImport from '../../src'
```

Comment:
@emmenko I tried changing this with `NODE_PATH` and to require this module, I have to do this
```js
import ProductTypeImport from '.'
```
which is very confusing